expats_static/deposit_uob.js

```diff
@@ -0,0 +1,19 @@
+  var requestOptions = {
+    method: 'GET',
+    redirect: 'follow'
+  };
+
+  fetch("http://localhost:3000/apideposit", requestOptions)
+    .then(response => response.text())
+    .then(result => {
+      depositARR = JSON.parse(result);
+      depositARR.forEach((item) => {
+        if (item.bank == "uob") {
+          document.getElementById('api_bank').innerHTML = `<img src="/expats-app/images/${item.bank}.png" class="img-fluid" alt="${item.bank}" width="200" height="200">`;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Geofflac/DevOps_ShowAndTell/pullRequest?prid=8167659)
```suggestion
          document.getElementById("api_bank").innerHTML = `<img src="/expats-app/images/${item.bank}.png" class="img-fluid" alt="${item.bank}" width="200" height="200">`;
```